network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/custom_hero_guts.html

```diff
@@ -72,13 +72,13 @@ <h2 class="tw-mt-4 tw-body-large">
 
   {% if download_file %}
 
-  {% with btn_setting=page.download_button_style primary_icon_class="tw-invert dark:group-hover:tw-invert-0" secondary_icon_class="group-hover:tw-invert dark:tw-invert dark:group-hover:tw-invert-0" tertiary_btn_class="tw-text-black dark:tw-text-white tw-font-light tw-underline hover:tw-opacity-50" %}
+  {% with btn_setting=page.download_button_style primary_icon_class="tw-invert dark:group-hover:tw-invert-0" secondary_icon_class="group-hover:tw-invert dark:tw-invert dark:group-hover:tw-invert-0" tertiary_btn_class="tw-text-black dark:tw-text-white tw-font-light tw-underline hover:tw-opacity-50" tertiary_icon_class="dark:tw-brightness-100 " %}
```

Comment:
This is a fix for a bug I found where dark mode icon for the primary and secondary buttons would not render correctly.